### ISSUE TITLE
7.8 No Servers Showing in Recommedned or Browser or Search

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#9999]
No servers are showing in Recommended or the Browser.
Tried resetting filters and empty searches which usually solve this
Deleted %appdata%
Restarted Steam
Still no servers.

**Comment by domenictiemo** [comment]
same problem like pamduncan
No servers are showing in Recommended or the Browser.
Tried resetting filters and empty searches which usually solve this
Deleted %appdata%
Restarted Steam
Still no servers.

**Comment by alphaest** [comment]
Same issue. List is empty.

**Comment by GriffIngalls** [comment]
I'm a Steam client also.  I can't see any servers using either 7.8 or 7.7.2.  The in-game server browser list is empty, like no servers are out there at all.

**Comment by pamduncan** [comment]
If you upgraded from Eco client 5.7 to to 7.8 and then rolled back due to issues with  7.8 the browser will not work unless you shutdown Eco, go to %appsdata%;/LocalLow/StrangeLoopGames/Eco folder.  Once you do that you can then start Eco back again and it will populate the appsdata with the right files you need and the servers should show up again

**Comment by Domino81** [comment]
Also can't see servers.  Was playing on a server, logged out briefly (a matter of minutes, less than three) and immediately couldn't see servers in the list.  Could see the one I was playing under "Your Worlds", but failed to connect.  Relaunched Eco, no effect.  Deleted AppData, verified files, reinstalled, rebooted - none of it worked.  Tried connecting to the server through an invite from someone in steam, no effect, failed to connect.  Server list still blank, unable to join previous server

**Comment by Jepps0n** [comment]
Don't know why its labeled ''fixed''. Have the same issue. Anyone with a solution? (and don't say delete  %appsdata%;/LocalLow/StrangeLoopGames/Eco folder) done a hundred times.

**Comment by Chronsy** [comment]
OK  The problem still persists for lots of people so not sure why it was marked as fixed  however - I was able to replicate the problem and also how to get back in to the server multiple times now..  I am having the same connectivity issues as most..  Nothing on the server Browser Can't join anything in Your Worlds..   

I downgraded my steam ECO client to an early beta, and then re upgraded to the 7.8 client.. Browse is still blank and nothing shows up on the main splash page as an active server.  However  when I chose Your Worlds there were now two listings for the server I play on "Doomland"  One showing a ping..  I was able to log in to it and play.  

HOWEVER if I log out for any reason I have to repeat the above process or I won't be able to connect..   This tells me that When I connect I am pulling some snippet of code from the master server that is borking things.  I was able to repeat this process 5 times and now have 5 different listings for Doomland on the Your Worlds Page.     

FYI this problem started happening to me a few hours BEFORE the 7.8 Update
